specs/order_spec.rb

```diff
@@ -117,15 +117,20 @@
 describe "Order Wave 2" do
   describe "Order.all" do
     it "Returns an array of all orders" do
-      # TODO: Your test code here!
+      orders = Order.all
+
+      expect(orders.length).must_equal 100
+      orders.each do |c|
+        expect(c).must_be_kind_of Order
+      end
```

Comment:
I don't know that I like `c` as the name of this iteration variable. `order` or at least `o` would be a better choice.